aict_tools/scripts/apply_disp_regressor.py

```diff
@@ -141,10 +140,11 @@ def main(
                 )
 
         elif config.data_format == "simple":
```

Comment:
The simple data format can also use `CTA` coordinate transforms. So you need to check for `coordinate_transforms` inside of `data_format == "simple"`